cmd/podman/containers/stop.go

```diff
@@ -124,6 +128,10 @@ func stop(cmd *cobra.Command, args []string) error {
 		stopOptions.Filters[split[0]] = append(stopOptions.Filters[split[0]], split[1])
 	}
 
+	// If no args, then nothing to stop
+	if len(args) == 0 {
```

Comment:
This breaks `stop -a`

Reply:
yikes.

Reply:
Just going to remove it and rely on the function handling it.